twtxt/cache.py

```diff
@@ -0,0 +1,66 @@
+"""
+    twtxt.cache
+    ~~~~~~~~~~~
+
+    This module implements a caching system for storing Tweets.
+
+    :copyright: (c) 2016 by buckket.
+    :license: MIT, see LICENSE for more details.
+"""
+
+import os
+import shelve
+
+import click
+
+
+class Cache:
+    cache_dir = click.get_app_dir("twtxt")
+    cache_name = "cache"
+
+    def __init__(self, cache_file, cache):
+        self.cache_file = cache_file
+        self.cache = cache
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.cache.close()
+
+    @classmethod
+    def from_file(cls, file):
+        """Try loading given cache file."""
+        try:
+            cache = shelve.open(file)
+            return cls(file, cache)
+        except:
+            raise ValueError("Cache file is invalid or can’t be accessed.")
```

Comment:
That's usually an OSError - Why not include original exception? 
Or don't even catch this one..